drivers/pwm/Kconfig.nrfx

```diff
@@ -0,0 +1,328 @@
+# Copyright (c) 2017 - 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig PWM_NRFX
+	bool "nRF PWM nrfx drivers"
+	depends on SOC_SERIES_NRF52X
+	help
+	  Enable support for nrfx Hardware PWM drivers for nRF52 MCU series.
+
+if PWM_NRFX
+
+config PWM_0_NRFX_PWM
+	bool "nRF HW PWM 0"
+	depends on SOC_SERIES_NRF52X
+	select NRFX_PWM
+	help
+	  Enable nRF Hardware PWM on port 0.
+
+if PWM_0_NRFX_PWM
+
+config PWM_0_NRFX_DEV_NAME
+	string "Nordic Semiconductor nRF52 series H/W PWM port  Device Name"
+	default "PWM_0"
+	help
+	  Specify the device name for the Nordic Semiconductor nRF52 series HW
+	  PWM port 0 driver.
+
+config PWM_0_NRFX_CLOCK_PRESCALER
+	int "Clock prescaler"
+	range 0 7
+	default 0
+	help
+		Clock = 16MHz / (2^prescaler)
+
+config PWM_0_NRFX_CH0_PIN
+	int "CH0 pin number"
+	range -1 47 if SOC_NRF52840_QIAA
+	range -1 31
+	default -1
+	help
+	  GPIO pin number to use for CH0
```

Comment:
On first read I wondered why the range starts from -1; guessing that is the value for 'not used'

Perhaps an extra "(set to -1 if not used)" in the comment would be nice here.

Alternatively, could you have an enable option for each pin, and then another option to set the range.